### ISSUE TITLE
Ys VIII: Lacrimosa of Dana

### ISSUE THREAD

**Issue by HorrorTroll** [Ryujinx/Ryujinx-Games-List#190]
#### Current on `master` :

* Build Version : 1.0.2866

```
Unhandled Exception: Ryujinx.HLE.Exceptions.ServiceNotImplementedException: Ryujinx.HLE.HOS.Services.Fspsrv.IFileSystemProxy: 22
```

After that it show loading screen, and then deadlock. But log screen still spaming:
```
00:05:52.769 | 0021 ServiceLm Log: Guest log:
 Log level: Info
 Message: 't'

00:05:52.791 | 0039 ServiceLm Log: Guest log:
 Log level: Info
 Time: 349s
 Line: 0
 Filename: ''
 Function: ''
 Module: ''
 Thread: 'DrawThread'
 Message: 'assertion error(C:/Users/ES-0033/Documents/WorkSpace/yen/projects/src/libmisc/libnx.cpp 2964) : m_BackBufferId != m_windowTarge'
```

#### Screenshots :
![image](https://user-images.githubusercontent.com/24840279/61586123-c6302980-ab96-11e9-888d-3d4315e69a3c.png)

#### Outstanding Issues:
- https://github.com/Ryujinx/Ryujinx/issues/639

#### Log file :
[Ryujinx.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/3414179/Ryujinx.log)

**Comment by GoffyDude** [comment]
## Ys VIII Lacrimosa of Dana

#### Current on `master` : 1.0.4481

Sits forever at loading screen spamming the below idletime message.

```
ServiceAm SetIdleTimeDetectionExtension: Stubbed. _idleTimeDetectionExtension: 1
```

#### Screenshots :

![2020-05-04 13_51_18-Ryujinx 1 0 4481 - Ys VIII_ Lacrimosa of DANA v1 0 0 (01007F200B0C0000) (64-bit)](https://user-images.githubusercontent.com/62343878/81007851-2daa2380-8e0f-11ea-8061-2c28ffff631c.png)

#### Log file :

[YsVIIILoD.zip](https://github.com/Ryujinx/Ryujinx-Games-List/files/4577097/YsVIIILoD.zip)

**Comment by B3astTech** [comment]
Updated

**Comment by EmulationFanatic** [comment]
Same status on 1.0.6432

**Comment by mmkzer0** [comment]
**Current on master : 1.0.6551**

Game loads and goes in-game, although with noticable graphical bugs, such as graphics loading in and out, character models not displaying properly all the time and blank dialogue screens. UI however loads fine. I will play to see if it crashes at any point futher into the game.

Console spams _HLE.OsThread.5 ServiceAm SetIdleTimeDetectionExtension: Stubbed. {idleTimeDetectionExtension: 1}_

Hardware Specs :

CPU: AMD Ryzen 5 3600 

GPU: AMD Radeon RX 5700 XT

RAM: 16GB

Screenshots :

![1](https://user-images.githubusercontent.com/74553440/107967522-2373a300-6fa5-11eb-847b-e226016e8295.PNG)

![2](https://user-images.githubusercontent.com/74553440/107967569-325a5580-6fa5-11eb-8e21-7023caa8f8da.PNG)

![3](https://user-images.githubusercontent.com/74553440/107967583-35eddc80-6fa5-11eb-8717-a3fe540db011.PNG)

![4](https://user-images.githubusercontent.com/74553440/107967597-39816380-6fa5-11eb-91a4-13d336c7d248.PNG)

![5](https://user-images.githubusercontent.com/74553440/107967600-3a19fa00-6fa5-11eb-8103-25021f87f270.PNG)

![6](https://user-images.githubusercontent.com/74553440/107967616-400fdb00-6fa5-11eb-8e4b-dc0fdaec3c3a.PNG)

Log File:

[YsVIIILoD.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/5982651/YsVIIILoD.log)

**Comment by EmulationFanatic** [comment]
Thanks for the test!

**Comment by EmulationFanatic** [comment]
@mmkzer0 can you re-test?

**Comment by mmkzer0** [comment]
I can. Am going to test it tomorrow with the newest available version.

**Comment by ConnorKolan** [comment]
I have a new Problem, is this issue still open or should I make a new thread ?

**Comment by EmulationFanatic** [comment]
> I have a new Problem, is this issue still open or should I make a new thread ?

These issues stay open as a compatibility guide. Go ahead and post the issue you're having.

**Comment by EmulationFanatic** [comment]
Updated

**Comment by ConnorKolan** [comment]
Nvm its the same problem, but I could find a solution in the comments.
https://youtu.be/YZTJrhIgCu8

[Ryujinx_1.1.248_2022-09-04_08-22-38.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/9484151/Ryujinx_1.1.248_2022-09-04_08-22-38.log)

Sry if the log is not readable, tell me if you need another

**Comment by ConnorKolan** [comment]
The purple text seems to be resolved by using opengl

**Comment by EmulationFanatic** [comment]
> The purple text seems to be resolved by using opengl

This is already in the OP.

**Comment by ConnorKolan** [comment]
You're right, but is there a solution for the problem with the blackscreen ?

**Comment by EmulationFanatic** [comment]
> You're right, but is there a solution for the problem with the blackscreen ?

Nothing for you to do currently. It needs to be fixed in the emulator.

**Comment by ConnorKolan** [comment]
Welp, thank you still.

**Comment by EmulationFanatic** [comment]
> Welp, thank you still.

Latest update fixes missing graphics in this game. Feel free to double check.

**Comment by EmulationFanatic** [comment]
Updated

**Comment by fann1993814** [comment]
Game Update Version : 1.0.0

Build Version : 1.1.883

Graphics Backend : Vulkan

The background sky in gaming cannot be displayed, it is always all black. In general, it should be rendered. 

Hardware Specs :

CPU: Mac M1 Air 

GPU: Mac M1 Air

RAM: 8GB

Screenshots :

![snapshot](https://github.com/Ryujinx/Ryujinx-Games-List/assets/21119633/5a614266-f4b9-41b2-9b5e-e2c37e6586ad)

Log File:

[Ryujinx_1.1.883_2023-06-11_17-58-55.log](https://github.com/Ryujinx/Ryujinx-Games-List/files/12266708/Ryujinx_1.1.883_2023-06-11_17-58-55.log)
